File: pyrb/Container/2.rb
#!/usr/bin/ruby
puts <<EOS
Content-type: text/html

EOS

al = ['A','B','C','D']
puts(al.length)
al.push('E')
print(al)
al.delete_at(0)
print("\n")
print(al)
print("\n")
del(al[0])
print(al)

File: pyrb/String/2.py
#!/usr/bin/python
import cgi;
import cgitb;cgitb.enable()
print("Content-Type: text/html\n")

print('hello world'.capitalize())
print('hello world'.upper())
print('hello world'.__len__())
print(len('hello world'))
print('hello world'.replace('world','programming'))